src/main/java/com/microsoft/bingads/internal/HttpClientWebServiceCaller.java

```diff
@@ -37,4 +45,41 @@ public HttpResponse post(URL requestUrl, List<NameValuePair> formValues) throws
 
     }
 
+    
+    private DefaultHttpClient createHttpClientWithProxy() {
+
+    	try {
+    		String webProxyEnabled = System.getProperty("web.proxy.enabled");
```

Comment:
also, instead of getting proxy data manually, is it possible to use ProxySelector similarly to this?

        AbstractHttpClient client = new DefaultHttpClient();

        HttpRoutePlanner  planner = new ProxySelectorRoutePlanner(client.getConnectionManager().getSchemeRegistry(), ProxySelector.getDefault());

        client.setRoutePlanner(planner);